lib/helpers.bash

```diff
@@ -227,6 +230,21 @@ _bash-it-migrate() {
   fi
 }
 
+_bash-it-version() {
+  _about 'shows current Bash-it version'
+  _group 'lib'
+
+  cd "${BASH_IT}" || return
+
+  echo "Current git SHA:"
+  echo "$(git log --pretty=format:'%h on %aI' -n 1)"
+
+  CURRENT_GIT_SHA=$(git rev-parse --verify HEAD)
+  echo "https://github.com/Bash-it/bash-it/commit/$CURRENT_GIT_SHA"
```

Comment:
What about forks in this case? Does it make sense to always refer to the `Bash-it/bash-it` location?

Reply:
Likely not. I can add some logic to handle this, it incurs another subshell to `git` - but from some naive timings, the entire command lasts less than 200ms, since I'm replacing the other subshell.